README.MD

```diff
@@ -350,6 +350,31 @@ export const plugins = {
 | enabled|bool|whether the bulk action toolbar should be used|
 | actions|arrayOf(object)|the actions (including button text, and event handler) that will be displayed in the bar|
 
+
+## Row renderer
+```javascript
+export const plugins = {
+    ROW: {
+        enabled: true,
+        renderer: ({rowProps, cells, row}) => {
+          return (
+              <tr { ...rowProps }>
+                  { cells }
+              </tr>
+            );
+        }
+    }
+};
+```
+| Prop|Type|Description|
+|:---------|:------------|:--------------|
+| enabled|bool|whether the bulk action toolbar should be used|
```

Comment:
Only need to list `enabled` and `renderer` here. I will add a new doc to discuss this feature in more detail.